site/content/api/Tenant-API.md

```diff
@@ -3,7 +3,10 @@ title = "Tenant API"
 weight = 410
```

Comment:
please change this to `407` so the page gets positioned in between the `Authentication` and `Device Registration` APIs.

Reply:
fixed